### ISSUE TITLE
cpp libtorch  transformerimpl lack some parameter between with python pytorch

### ISSUE THREAD

**Issue by mullerhai** [pytorch/pytorch#147581]
### 🐛 Describe the bug

Hi,
I find  libtorch some layer impl not the same as python pytorch  ,like transformer layer

https://pytorch.org/docs/stable/generated/torch.nn.Transformer.html

in python 
Transformer
CLASStorch.nn.Transformer(d_model=512, nhead=8, num_encoder_layers=6, num_decoder_layers=6, dim_feedforward=2048, dropout=0.1, activation=<function relu>, custom_encoder=None, custom_decoder=None, layer_norm_eps=1e-05, batch_first=False, norm_first=False, bias=True, device=None, dtype=None)

in cpp  TransformerImpl  not have   [layer_norm_eps=1e-05, batch_first=False, norm_first=False, ] ,three parameter not supply ,so how to pass these parameter in cpp

thanks

### Versions

libtorch latest pytorch 2.6
